src/pages/publication/PublicationList.tsx

```diff
@@ -17,12 +19,20 @@ const StyledTable = styled(Table)`
 `;
 
 const StyledTableCellForStatus = styled(TableCell)`
-  width: 15%;
+  width: 10%;
 `;
 const StyledTableCellForDate = styled(TableCell)`
   width: 15%;
 `;
 
+const StyledEditIcon = styled(EditIcon)`
+  margin-right: 0.5rem;
+`;
+
+const StyledDeleteIcon = styled(DeleteIcon)`
+  margin-right: 0.5rem;
+`;
```

Comment:
kunne lagd en `StyledText` som satte `margin-left` for teksten, så hadde du sluppet 2x styled-components :)

Reply:
evt lagd knappe-komponenter. Vi får se om det blir mange av dem